.env

```diff
@@ -1,4 +1,7 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```

Comment:
This file shouldn't be committed, seems like the automated script to stop it committing didn't run. Could you run the following: 
`git checkout origin/latest -- .env`
Then commit and push the changes, then run:
`git update-index --assume-unchanged .env`

Then